studies/templates/emails/base.html

```diff
@@ -3,3 +3,4 @@
 <br />
 <a href="{{ base_url }}{% url 'web:email-preferences' %}">Update your email preferences</a>
 <a href="{{ base_url }}{% url 'web:email-unsubscribe-link' username=username token=token %}">Unsubscribe from all emails</a>
```

Comment:
I think this PR should also add "CHS" to the linked text in these lines ("Update your CHS email preferences", "Unsubscribe from all CHS emails"), since it's a related change that probably doesn't need a separate PR. More info in the issue.